gpytorch/kernels/kernel.py

```diff
@@ -102,14 +102,14 @@ def __init__(
         self.ard_num_dims = ard_num_dims
         self.batch_size = batch_size
         self.__has_lengthscale = has_lengthscale
-        self.param_transform = param_transform
         if has_lengthscale:
             self.eps = eps
             lengthscale_num_dims = 1 if ard_num_dims is None else ard_num_dims
             self.register_parameter(
                 name="log_lengthscale",
                 parameter=torch.nn.Parameter(torch.zeros(batch_size, 1, lengthscale_num_dims)),
                 prior=log_lengthscale_prior,
```

Comment:
As mentioned in a previous PR, we'll probably want to soft-deprecate `log_lengthscale_prior` in favor of `lengthscale_prior`.

Reply:
yeah happy to do this as part of this PR